dist/ecommerce.js

```diff
@@ -30,7 +27,18 @@ return Promise.all([
     }
   })
 ]).then(([AndroidAppLinks, iOSUniveralLinks]) => {
-  return JSON.stringify({AndroidAppLinks, iOSUniveralLinks});
+  return {
+    AndroidAppLinks,
+    iOSUniveralLinks
+  };
 }).catch(error => {
-  return JSON.stringify({message: error.message, error: error});
+  return {
+    message: error.message,
+    error: error
+  };
+}).then(result => {
+  return {
+    ...result,
+    shopify: getShopifyMetadata()
```

Comment:
Could capitalise this to keep with the global name (and the fact that Android and iOS keys are capitalised).

```suggestion
    Shopify: getShopifyMetadata()
```